websites/T/TabNews/presence.ts

```diff
@@ -0,0 +1,97 @@
+const presence = new Presence({
+		clientId: "1083051669996187748",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Outros",
+			largeImageKey: "https://i.imgur.com/pQeO9sn.png",
+			startTimestamp: browsingTimestamp,
+			buttons: [
+				{
+					label: "Ver conteúdo",
+					url: document.location.href,
+				},
+			],
```

Comment:
Buttons that can lead to home pages are [not allowed](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets). Generally, buttons should lead to more specific content.
```suggestion
```